### DISCUSSION TITLE
Add Notion-like calendar/schedule

### DISCUSSION THREAD

**Discussion by eve0415** [outline/outline#7405] [Ideas]
<img width="765" alt="image" src="https://github.com/user-attachments/assets/1bc7a48f-dba4-4045-86ca-ef6dfc7288ca">

<img width="753" alt="image" src="https://github.com/user-attachments/assets/d26715e8-c5b3-4458-817f-c024c73a3be1">

**Comment by D1reW0lf** [comment]
And, if one can dream, with the ability to export as a caldav / ics to sync with the other calendar apps.
